options.go

```diff
@@ -47,9 +47,14 @@ type RequestOptions struct {
 
 // TransportOptions are transport related options.
 type TransportOptions struct {
-	ServiceName  string   `short:"s" long:"service" description:"The TChannel/Hyperbahn service name"`
-	HostPorts    []string `short:"p" long:"peer" description:"The host:port of the service to call"`
-	HostPortFile string   `short:"P" long:"peer-list" description:"Path of a JSON file containing a list of host:ports"`
+	ServiceName      string            `short:"s" long:"service" description:"The TChannel/Hyperbahn service name"`
+	HostPorts        []string          `short:"p" long:"peer" description:"The host:port of the service to call"`
+	HostPortFile     string            `short:"P" long:"peer-list" description:"Path of a JSON file containing a list of host:ports"`
+	CallerOverride   string            `long:"caller" description:"Caller will override the default caller name (which is yab-$USER)."`
+	TransportOptions map[string]string `long:"topt" description:"Custom options for the specific transport being used"`
```

Comment:
Discourage, since you can skip some of the protections (like no overriding caller when benchmarking) using these. I want the ability to override these low level params but don't necessarily want them used in the common cases.